samples/Server/src/lib/chat/userIdToUserConfigMap.ts

```diff
@@ -7,4 +7,6 @@ interface UserConfig {
   id: string;
 }
 
+// This is for samples only.
+// All User information (name, profile image) are stored in memory only and deleted after refresh
```

Comment:
nitty nit: "refresh" might be confusing here. How about "when the server restarts".

Should we link to some potential storage technologies, perhaps on azure? wdyt @alkwa-msft ?

Reply:
Updated :)

Reply:
The idea is to keep it simple.

"For the purpose of this sample, we opted to use an in-memory data store. This means that if the web application is restarted any information maintained would be wiped. For longer term storage solutions we suggest using [Azure Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction)

Reply:
Updated comments

Reply:
We should probably be telling customers to use an Azure database (https://azure.microsoft.com/en-ca/product-categories/databases/) instead of a file storage as this has many read writes

Reply:
Heh, I had suggested yesterday that we link to https://docs.microsoft.com/en-us/azure/architecture/guide/technology-choices/data-store-decision-tree ( let Azure recommend what to use ;) )

Reply:
Oh yeah that's way better ^

Reply:
So should I update the link in the comments to the decision tree one?

Reply:
Updated the comment, do you mind having a quick look?